misc/ExampleExtension.py

```diff
@@ -0,0 +1,76 @@
+import bpy
+import sys
+from io_scene_gltf2.io.com.gltf2_io_extensions import Extension
```

Comment:
You can't import from `io_scene_gltf2` at the top level because you can't control in what order the addons get loaded and it might not exist yet. You'll need to do something like this

````py
class glTF2UserExtension:
    def __init__(self):
        import io_scene_gltf2
        self.io_scene_gltf2 = io_scene_gltf2

    def export(...):
        ext = self.io_scene_gltf2.Extension(...)
````

That's what I meant about the user-extensions needing to wait until the `Gltf2UserExt` class is called into to import `io_scene_gltf2`.

(If you want to test it, disable the glTF2 addon, restart, and then enable it to make sure it gets loaded last.)